src/ProjectOrigin.WalletSystem.Server/CommandHandlers/TransferCertificateCommandHandler.cs

```diff
@@ -53,7 +54,7 @@ public async Task Consume(ConsumeContext<TransferCertificateCommand> context)
             List<Task> tasks = new();
             foreach (var slice in reservedSlices)
             {
-                var builder = new RoutingSlipBuilder(NewId.NextGuid());
+                var builder = new RoutingSlipBuilder(msg.TransferRequestId);
```

Comment:
Why this change?

Reply:
Since the ID is now generated earlier in the flow, in the controller, so it can be returned to the caller.